### PR TITLE
Avoid glitch on checkbox click

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.5.x
| Description?  | change in the product grid, in position mode, the mousedown target to the proper column to avoid glitch on checkbox click (the mousedown on the whole row didn't make sense anyway)
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/10319
| How to test?  | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11071)

### CODE REVIEW

File: admin-dev/themes/default/js/bundle/product/catalog.js [hunk 1]
```diff
@@ -71,8 +71,9 @@ $(document).ready(function() {
 	 * Sortable case when ordered by position ASC
 	 */
 
-	$("body").on("mousedown", "tbody.sortable [data-uniturl]", function () {
-		$(this).find('input:checkbox[name="bulk_action_selected_products[]"]').attr("checked", true);
+	$("body").on("mousedown", "tbody.sortable [data-uniturl] td.placeholder", function () {
+	    const trParent = $(this).closest('tr');
```
Comment (PierreRambaud) [inline admin-dev/themes/default/js/bundle/product/catalog.js]: No need to use const here, you never reuse the variable.
Watch out with indentation, sometimes 2, 4 and even 6 :)

**Comment by jolelievre** [comment]
thanks @PierreRambaud I used a var, and I autoindented the whole file because it was messy :P

**Review by eternoendless — CHANGES_REQUESTED** [review]
Rules for indentation is 2 spaces for Javacript files. See [code conventions](https://devdocs.prestashop.com/1.7/development/coding-standards/#javascript-code-conventions).

Also, please fill out the table properly.

**Comment by jolelievre** [comment]
my bad, I didn't pay attention to the branch, I changed the autoindent rules, so it's two spaces now

**Review by eternoendless — APPROVED** [review]
LGTM

**Comment by PierreRambaud** [comment]
Thanks @jolelievre

**Pull request merged**